packages/xstate-react/src/useService.ts

```diff
@@ -17,7 +17,7 @@ export function fromService<TContext, TEvent extends EventObject>(
     send: service.send.bind(service),
     subscribe: service.subscribe.bind(service),
     stop: service.stop!,
-    current: service.initialized ? service.state : machine.initialState,
+    current: service._status !== 0 ? service.state : machine.initialState,
```

Comment:
Should probably import `InterpreterStatus` here

Reply:
couldn't do this right now, because the older XState versions don't have such export - I've exported `InterpreterStatus` from the `xstate` package though and added a comment here that we should migrate to it whenever we decide to release a new major of `@xstate/react`